gpt4all-chat/qml/HomeView.qml

```diff
@@ -189,6 +189,9 @@ Rectangle {
                                     Accessible.role: Accessible.Paragraph
                                     Accessible.name: qsTr("Latest news")
                                     Accessible.description: qsTr("Latest news from GPT4All")
+                                    onLinkActivated: function(link) {
+                                        Qt.openUrlExternally(link);
+                                    }
```

Comment:
For consistency, I think we should either remove the MouseArea for links in settings (MySettingsLabel), or use a MouseArea in similar places such as this one. I lean towards the former.